server/pages/index.js

```diff
@@ -22,28 +22,29 @@ import { createReduxStore } from 'state';
 
 const debug = debugFactory( 'calypso:pages' );
 
-var HASH_LENGTH = 10,
+let HASH_LENGTH = 10,
 	URL_BASE_PATH = '/calypso',
 	SERVER_BASE_PATH = '/public',
 	CALYPSO_ENV = process.env.CALYPSO_ENV || process.env.NODE_ENV || 'development';
 
-var staticFiles = [
+const staticFiles = [
```

Comment:
Unrelated, but many of these `staticFiles` are large and their contents are loaded synchronously during the hash step for `getUrl`. Would be nice to have some parallelization there.